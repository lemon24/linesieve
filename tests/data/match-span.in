--section '(\S+):$'
match-span -s ^simple$ --start 2 --end 4
match-span -s ^simple-invert$ --start 2 --end 4 -v
match-span -s ^no-end$ --start 2
match-span -s ^no-end-invert$ --start 2 -v
match-span -s ^no-start$ --end 4
match-span -s ^no-start-invert$ --end 4 -v
match-span -s ^repl$ --start 2 --end 4 --repl ...
match-span -s ^repl-invert$ --start 2 --end 4 --repl ... -v
match-span -s ^repl-edges$ --start 1 --end 5 --repl ...
match-span -s ^repl-invert-one$ --start 2 --end 3 --repl ... -v
match-span -s ^repl-no-end$ --start 2  --repl ...
match-span -s ^repl-two-ends$ --start 3 --end [45] --repl ...
match-span -s ^repl-invert-two-starts$ --start [23] --end 5 --repl ... -v
match-span -s ^repl-capture$ --start '(2)' --end 4 --repl '>\\<'
match-span -s ^repl-invert-capture$ --start '(2)' --end 4 --repl '>\1<' -v

---
simple:
1
2
3
4
5
simple-invert:
1
2
3
4
5
no-end:
1
2
3
4
5
no-end-invert:
1
2
3
4
5
no-start:
1
2
3
4
5
no-start-invert:
1
2
3
4
5
repl:
1
2
3
4
5
repl-invert:
1
2
3
4
5
repl-edges:
1
2
3
4
5
repl-invert-one:
1
2
3
4
5
repl-no-end:
1
2
3
4
5
repl-no-start-invert:
1
2
3
4
5
repl-two-ends:
1
2
3
4
5
repl-invert-two-starts:
1
2
3
4
5
repl-capture:
1
2
3
4
5
repl-invert-capture:
1
2
3
4
5
