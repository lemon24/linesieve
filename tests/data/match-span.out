simple
2
3
simple-invert
1
4
5
no-end
2
3
4
5
no-end-invert
1
no-start
no-start-invert
1
2
3
4
5
repl
...
2
3
...
repl-invert
1
...
4
5
repl-edges
1
2
3
4
...
repl-invert-one
1
...
3
4
5
repl-no-end
...
2
3
4
5
repl-no-start-invert
1
2
3
4
5
repl-two-ends
...
3
...
repl-invert-two-starts
1
...
5
repl-capture
>\<
2
3
>\<
repl-invert-capture
1
>2<
4
5
